storage/mock/storedDataFactoryStub.go

```diff
@@ -0,0 +1,20 @@
+package mock
```

Comment:
Please move this somewhere in the testscommon instead.

Reply:
moved in testscommon/storage package